src/js/utils/faye.js

```diff
@@ -124,6 +125,8 @@ export function handleUserSubscription({appUser, event}) {
                 }, ANIMATION_TIMINGS.PAGE_TRANSITION);
             }, ANIMATION_TIMINGS.PAGE_TRANSITION);
         }
+    } else if (event.type === 'link:cancelled') {
+        return cancelTwilioLink();
```

Comment:
I would check the platform before saying it's twilio being cancelled. Sure we only support that for now, but we don't know what the future holds for us and on which version of the widget people will be stuck when we'll roll out new integrations.